### ISSUE TITLE
Line length to 120

### ISSUE THREAD

**Issue by soffes** [realm/SwiftLint#28]
On a 27-inch display, you can comfortable fit two 120 wide files at 14pt with sidebars and all. On a 13" rMBP, you can fit one. I've had my guide set at 120 for years. Personally, I think 100 is overly narrow. Almost all of the warnings for one of my projects was length being over 100 but less 120.

I'm guessing there's a reason you picked 100 though. I'd love to know why!

I'm tempted to ask for a way to configure it, but I think one of the biggest benefits of this is we can all start doing things the same as a community.

**Comment by segiddins** [comment]
This should definitely be configurable, but I personally use 100 because you can easily fit two documents side-by-side without wrapping on a reasonable laptop screen.

**Comment by aarondaub** [comment]
Perhaps we should add the ability to configure this (issue for doing so in general: #26) and then add a standardized YAML file that we include by default?

**Comment by soffes** [comment]
Personally, I'd rather see JSON. I feel like I only see YAML in the Ruby world. I feel like more iOS developer would be familiar with JSON than YAML.

**Comment by segiddins** [comment]
@soffes please take that discussion to https://github.com/realm/SwiftLint/issues/3, and keep in mind that YAML is a strict superset of JSON.

**Comment by soffes** [comment]
That unrelated to if the target audience is familiar with it.

**Comment by jpsim** [comment]
> I'm guessing there's a reason you picked 100 though. I'd love to know why!

I picked the initial defaults somewhat haphazardly, and definitely appreciate issues being created to suggest more sensible defaults.

Specifically, I picked 100 for line length because that's what I have my Xcode page guide set to, but I *do* find myself going up to 120 quite often.

Ultimately, this is a question of personal preference so no matter what we pick, it'll be "wrong" for some people/projects.
